system/lib/pthread/pthread_create.c

```diff
@@ -139,14 +139,18 @@ int __pthread_create(pthread_t* restrict res,
     new->stack_owned = 1;
   }
 
+  if (!libc.threads_minus_1++) libc.need_locks = 1;
```

Comment:
can this not be moved below the `--` a few lines below, to avoid that `--`? Or can the code in the middle need locking? (if so maybe a comment?)

Reply:
Done